src/DependencyInjection/SonataClassificationExtension.php

```diff
@@ -160,6 +160,7 @@ public function registerDoctrineMapping(array $config)
                 [
                     'name' => 'context',
                     'referencedColumnName' => 'id',
+                    'nullable' => false,
```

Comment:
I hoped that it is not BC =)
Db update is not necessary. But it there are records with nulled `contexts` exists, this will break their app. Do you think, i should remove this commit from PR?

Reply:
How is it not necesarry? If you have by default nullable true and we change it here, wouldn’t doctrine command report a problem? Or as you said if they have already null values. I think you should, is this really needed? What do rest of @sonata-project/contributors think?

Reply:
> I think you should, is this really needed?

I will comment lines with `'nullable' => false,` and i will add comment with `NEXT_MAJOR: uncomment this line, context should not be nullable`. Is this ok?

Reply:
or maybe just create a PR to master instead of leaving comments? also let's wait for someone else to comment

Reply:
@kunicmarko20 PR https://github.com/symfony/recipes-contrib/pull/388 with fix for MediaBundles was merged. And now with https://github.com/symfony/recipes-contrib/pull/389 ClassificatioBundle will work together with MediaBundle. So, may be lets disallow null value on foreign keys for master branch?

Reply:
yes, remove it from here and create a PR on master.

Reply:
I created new PR https://github.com/sonata-project/SonataClassificationBundle/pull/404 and updated this one